css/style.css

```diff
@@ -28,576 +28,585 @@ h2,
+  font-size: 12px;
+  line-height: 25px;
+  font-family: "Work Sans", sans-serif;
 }
 
 .header-right {
-    float: right;
+  float: right;
 }
 
 .header-right a:hover {
-    border-bottom: 3px #4a50bb solid;
+  border-bottom: 3px #4a50bb solid;
 }
 
 .header-right button {
-    border: 1px solid #4a50bb;
-    font-weight: 500;
-    outline: none;
-    background: #4a50bb;
-    color: #fff;
-    padding: 15px;
-    border-radius: 3px;
-    cursor: pointer;
+  border: 1px solid #4a50bb;
+  font-weight: 500;
+  outline: none;
+  background: #4a50bb;
+  color: #fff;
+  padding: 15px;
+  border-radius: 3px;
+  cursor: pointer;
 }
 
 @media screen and (max-width: 500px) {
-    .header a {
-        float: none;
-        display: block;
-        text-align: left;
-    }
-    .header-right {
-        float: none;
-    }
+  .header a {
+    float: none;
+    display: block;
+    text-align: left;
+  }
+  .header-right {
+    float: none;
+  }
 }
 
-
 /* OUR PROGRAM PART */
 
 .our-program {
-    display: flex;
-    -webkit-box-pack: center;
-    justify-content: center;
-    width: 100%;
-    padding: 50px 20px;
-    position: relative;
-    box-shadow: none;
-    background: none;
-    z-index: 1;
-    flex: 0 0 auto;
-    max-width: 1280px;
-    margin: auto;
+  display: flex;
+  -webkit-box-pack: center;
+  justify-content: center;
+  width: 100%;
+  padding: 50px 20px;
+  position: relative;
+  box-shadow: none;
+  background: none;
+  z-index: 1;
+  flex: 0 0 auto;
+  max-width: 1280px;
+  margin: auto;
 }
 
 .explanation-wrapper {
-    width: 100%;
-    display: flex;
-    margin: 50px 0px;
+  width: 100%;
+  display: flex;
+  margin: 50px 0px;
 }
 
 .explanation {
-    display: flex;
-    flex: 0 0 auto;
-    flex-direction: column;
-    align-items: flex-start;
-    -webkit-box-pack: start;
-    justify-content: flex-start;
-    padding: 0px 50px 0px 0px;
-    width: 50%;
-    order: 1;
+  display: flex;
+  flex: 0 0 auto;
+  flex-direction: column;
+  align-items: flex-start;
+  -webkit-box-pack: start;
+  justify-content: flex-start;
+  padding: 0px 50px 0px 0px;
+  width: 50%;
+  order: 1;
 }
 
 .op-title {
-    padding: 0;
-    font-size: 20px;
-    line-height: 140%;
-    margin-bottom: 20px;
-    text-transform: uppercase;
-    font-family: "Work Sans Bold", sans-serif;
-    font-weight: 1000;
-    color: rgb(28, 34, 148);
-    text-align: left;
-    max-width: none;
+  padding: 0;
+  font-size: 20px;
+  line-height: 140%;
+  margin-bottom: 20px;
+  text-transform: uppercase;
+  font-family: "Work Sans Bold", sans-serif;
+  font-weight: 1000;
+  color: rgb(28, 34, 148);
+  text-align: left;
+  max-width: none;
 }
 
 .program-info {
-    padding: 0;
-    color: rgb(45, 48, 55);
-    text-align: left;
-    max-width: 470px;
-    line-height: 160%;
-    font-size: 16px;
-    font-family: "Space Mono", monospace;
-    margin: 0px;
-    word-break: break-word;
-    font-weight: 300;
-    margin: 0px 0px 20px 0px;
+  padding: 0;
```

Comment:
This part is hasan's part. And i think (red-old one) program - info has many codes than (Green) New program-info. İf i merge this, do we lose the codes or not?